### DISCUSSION TITLE
CS%E5%9F%BA%E7%A1%80/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/%E7%BC%96%E7%A8%8B%E5%9F%BA%E7%A1%80/%E5%9F%BA%E7%A1%80%E7%BC%96%E7%A8%8B%E8%B5%84%E6%BA%90%E6%B1%87%E6%80%BB/

### DISCUSSION THREAD

**Discussion by giscus** [MingLLuo/CS-Plan#4] [Announcements]
# CS%E5%9F%BA%E7%A1%80/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/%E7%BC%96%E7%A8%8B%E5%9F%BA%E7%A1%80/%E5%9F%BA%E7%A1%80%E7%BC%96%E7%A8%8B%E8%B5%84%E6%BA%90%E6%B1%87%E6%80%BB/

https://cs-plan.com/CS%E5%9F%BA%E7%A1%80/%E8%AF%BE%E7%A8%8B%E6%8E%A8%E8%8D%90/%E7%BC%96%E7%A8%8B%E5%9F%BA%E7%A1%80/%E5%9F%BA%E7%A1%80%E7%BC%96%E7%A8%8B%E8%B5%84%E6%BA%90%E6%B1%87%E6%80%BB/

**Comment by kok-s0s** [comment]
不清楚 scnu 的大一新生学C++是否还在用谭浩强的书，如果有，建议少看。可以多敲敲这个[网页](https://www.cbyexample.com/)的示例代码，对学C/C++有点帮助。
